src/app/public/shared/card/card.component.html

```diff
@@ -12,8 +12,16 @@
   <td class="tableRepository__info">
     <tr>
       <div class="tableRepository__status">
-        <div class="cell"><app-status></app-status></div>
-        <div class="cell"><app-status></app-status></div>
+        <div class="cell">
```

Comment:
this should be `class="tableRepository__cell"` ????

Reply:
I'll change it here https://github.com/MariaAlba1306/github/pull/10